pkg/cmd/flags.go

```diff
@@ -276,12 +276,7 @@ func Content(flagval string) string {
 	content := flagval
 	if path, err := filepath.Abs(flagval); err == nil {
 		if _, err := os.Stat(path); err == nil {
-			// gosec flagged this:
-			// G304 (CWE-22): Potential file inclusion via variable
-			//
-			// Disabling as we require a user to configure their own environment.
-			/* #nosec */
-			if data, err := os.ReadFile(path); err == nil {
+			if data, err := os.ReadFile(path); err == nil /* #nosec */ {
```

Comment:
**NOTE:** Latest gosec (released today) changed the way `#nosec` is parsed. It has to be _on_ the affected line.